dvc/output/base.py

```diff
@@ -131,17 +128,30 @@ def changed_cache(self):
         return cache.changed_cache(self.checksum)
 
     def status(self):
-        if self.changed():
-            # FIXME better msgs
-            return {str(self): "changed"}
+        if self.checksum and self.use_cache and self.changed_cache():
+            return {str(self): "missing cache"}
```

Comment:
`not in cache`?

Reply:
I mean missing cache sounds like cached is missed itself to me

Reply:
Great point!